apps/snitch_core/priv/repo/migrations/20180314131025_add_unique_index_to_state.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Snitch.Repo.Migrations.AddUniqueIndexToState do
+  use Ecto.Migration
+
+  def change do
+    create(
+      unique_index(:snitch_states, [:abbr, :country_id], name: :snitch_state_abbr_and_country_id)
+    )
```

Comment:
Are you sure that this is the right way to add a unique index? Shouldn't that be:
 create unique_index(......) instead of create(unique_index(....))